### PR TITLE
Update package fix - 2

### PR DESCRIPTION
**Pull request by JeremyLecoeur**
# Description

Currently, the python library allow user to mess with the normal flow of state from the state machine.
We propose removing the state from the update function and also removing all the functions that mutate the state

## Github Issue

https://app.clickup.com/t/11h2q5

## Type of change

- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+
+## [2.13.1]
+### Changed
+- Fixed wrong tag for release + documentation about release
+
 ## [2.13.0]
 ### Removed
 - State of packages can no longer be set through this library 
+- Non functional. DO NOT USE
```
Comment (rlmv) [inline CHANGELOG.md]: You can remove this warning - technically you can check out this tag from Github and the code will still work, just not the version in PyPi.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Documentation at https://developer.blackfynn.io/python/
 
+
+## [2.13.1]
+### Changed
+- Fixed wrong tag for release + documentation about release
```
Comment (rlmv) [inline CHANGELOG.md]: For consistency with the rest of the changelog, can you split this out into 

```
### Fixed
- Incorrect version 

### Added
- Documentation describing the release process
```

File: CHANGELOG.md (hunk 1 above)
Comment (JeremyLecoeur) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (JeremyLecoeur) [reply CHANGELOG.md]: done

**Pull request merged**